drivers/sensor/Kconfig

```diff
@@ -22,6 +22,8 @@ config SENSOR_INIT_PRIORITY
 
 config SENSOR_SHELL
 	bool "Enable sensor shell"
+	depends on SHELL
+	default y
```

Comment:
why is this being enabled by default?

Reply:
it's enabled only when shell is enabled. I think it's good to expose shell commands to the user. Since ROM is the only cost and it's more likely that when shell is enabled then it is not an issue.

Same is done in the logger: https://github.com/zephyrproject-rtos/zephyr/blob/0fb370b257e1060c0155df30059e5177f07f8a62/subsys/logging/Kconfig#L276

Reply:
ok, I see this is enclosed within "If SENSOR" already.